### PR TITLE
Fix dom highlight dimensions after jquery upgrade

### PR DESCRIPTION
**Pull request by kuceb**
this restores element highlighting functionality to before the jquery upgrade
- dont use jquery for width/height calculations for dom highlights
- put opacity on wrapper element
- yes- its not perfect- but it's back to prev release functionality, and we should eventually use strategy outlined here #5281 

### User facing changelog
n/a

### How has the user experience changed?
**hightlight element opacity**
- before

- after

**highlight on element w/ css transform***
- before (completely off):

- after (still not perfect, but matches prev release):

**highlight on element w/ parent css transform***
- before (not great)

- after (still not great, but matches prev release)

### PR Tasks
- [x] add test for vanilla inline element

- [x] Have tests been added/updated?
- n/a Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- n/a Have API changes been updated in the [`type definitions`](cli/types/index.d.ts)?
- n/a Have new configuration options been added to the [`cypress.schema.json`](cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/runner/src/lib/dom.js [hunk 1]
```diff
@@ -71,6 +71,9 @@ function addElementBoxModelLayers ($el, $body) {
   const dimensions = getElementDimensions($el)
 
   const $container = $('<div class="__cypress-highlight">')
+  .css({
+    opacity: 0.7,
```
Comment (jennifer-shehane) [inline packages/runner/src/lib/dom.js]: Why the opacity change to a more opaque value? :D

Comment (Bkucera) [reply packages/runner/src/lib/dom.js]: opacity used to be on each of the three elements, so it was really 0.6 stacked three times. So 0.7 looked the most correct to me

**Comment by jennifer-shehane** [comment]
There's not an issue open for this? I thought I saw a screenshot somewhere explaining the problem of the highlighting. Can you show like, before/after? 😬 It's hard to tell what the original problem was from the description. 

I get it's like, a pre-release change, but supposedly the experience before this fix and after this fix should have changed and should still be detailed for PR reviewers.

**Comment by Bkucera** [comment]
@jennifer-shehane see updated PR comment for screenshots

**Comment by jennifer-shehane** [comment]
@Bkucera Thanks! This is so helpful 😍

**Review by jennifer-shehane — DISMISSED** [review]

**Review by brian-mann — APPROVED** [review]

**Pull request merged**